Projects/DesignSystem/Sources/TextField/SearchTextField.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  SearchTextField.swift
+//  DesignSystem
+//
+//  Created by Chanhee Jeong on 2022/10/25.
+//  Copyright © 2022 zesty. All rights reserved.
+//
+import UIKit
+import SnapKit
+
+public final class SearchTextField: UIView {
+    public var placeholder: String = "장소"
+    
+    public let textField = UITextField()
+    private let leftImage = UIImageView()
+    private let backgroundView = UIView()
+    private let cornerRadius: CGFloat = 45/2
+    
+    public init() {
+        super.init(frame: .zero)
+        configureUI()
+        createLayout()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+extension SearchTextField {
+    private func configureUI() {
+        textField.placeholder = placeholder
+        textField.clearButtonMode = .always
+        textField.returnKeyType = .go
+        textField.textColor = .black
+        textField.font = UIFont.systemFont(ofSize: 16, weight: .medium)
+        backgroundView.layer.borderWidth = 2
+        backgroundView.layer.cornerRadius = cornerRadius
+        backgroundView.clipsToBounds = true
+    }
+    
+    private func createLayout() {
+        addSubviews([backgroundView, textField])
+        
+        textField.snp.makeConstraints {
+            $0.verticalEdges.equalToSuperview().inset(12)
```

Comment:
p2;

텍스트필드 높이가 지정되어 있어서 inset은 필요 없어 보여요. 오히려 문제가 생길수도 있지 않을까 해서 그냥 중앙정렬 해줘도 될 것 같습니다!

```suggestion
            $0.centerY.equalToSuperview()
```

Reply:
반영